README.md

```diff
@@ -1,5 +1,5 @@
 # The SpatialOS Game Development Kit for Unreal (alpha)
```

Comment:
I might be being dim, but I can't find where this file gets displayed. It's quite different from the one [here](https://github.com/spatialos/UnrealGDK/blob/release/README.md) and it doesn't appear to be in the docs either.

Reply:
AAH, I've just changed branch to `docs-release` and it's in there, but it was last edited in April. I think this probably means the `release` branch readme is more accurate and up-to-date, so I guess I need to make changes to that by branching from the main `release` branch?

Reply:
I am enjoying this one-person conversation. It's very familiar to me.